containers/Makefile

```diff
@@ -83,7 +91,7 @@ endif
 $(info IMAGE_PREFIX=$(IMAGE_PREFIX))
 
 
-all: nginx-proxy nginx-storage concent-api postgresql verifier signing-service concent-signing-service-package geth
+all: nginx-storage concent-api postgresql verifier signing-service concent-signing-service-package geth nginx-proxy
```

Comment:
As discussed today - the order should not affect the build result. If it does, the problem is elsewhere.

From what you said it seems that the problem here is that building `nginx-proxy` results in `RELEASE-VERSION` being left in Concent source. Removing it would solve the problem.